### ISSUE TITLE
Plotting images on tensorboard

### ISSUE THREAD

**Issue by ParikaGoel** [HyoungsungKim/Pytorch-tensorboard_tutorial#2]
Hello,

I am trying to visualize images at each epoch. My code is as below:

self.writer.add_images('train_gt', target * input_mask)
self.writer.add_images('train_pred', output.detach() * input_mask)

I am training the model for 50 epochs and writing image to tensorboard on each epoch.
But there are only 10 images plotted in the tensorboard.

Could you please let me know where am I doing wrong ?
I would really appreciate any help.

**Comment by HyoungsungKim** [comment]
I am not sure because i cannot see your detail code.

But in my opinion, i recommend check your tag name again.

If you use same name, pictures will be overwritten by next epoch's pictures.

How about adding a specific number like 'train_gt' + str(epoch_number)

For example
```python
self.writer.add_images('train_gt' + str(epoch_num), target * input_mask)
```
I hope it is helpful for you :)

**Comment by ParikaGoel** [comment]
hi,

thanks for your quick reply.
I figured it out. I had to add epoch number as the third argument

self.writer.add_images('train_gt', target * input_mask,epoch)
